src/utils/users.ts

```diff
@@ -56,7 +56,7 @@ export const setDebouncedFilteredAssignees = (
   }
   const newTimeoutId = setTimeout(async () => {
     setLoading(true)
-    const newAssignees = await getAssigneeList(z.string().parse(token), newInputValue, 2000, '0') // 2000 is hardcoded for now
+    const newAssignees = await getAssigneeList(z.string().parse(token), newInputValue, 10000, '0') // 2000 is hardcoded for now
```

Comment:
will it hit any perfomance issue?

the comment still says 2000

Reply:
Having 10000 items in the payload will definitely increase the response time but since this is suppose to filter by keyword we won't have lot of payload so performance should be fine.

Removed comment in the commit below.

Reply:
Approved but please test this is load testing branch